oaknational/Oak-Web-Application
# feat: 1143 update copy on notification box on homepage - TO BE MERGED ON 09.01. BEFORE 9AM

## Description

Updated copy and link on the homepage notification box.

TO BE MERGED ON 09.01. BEFORE 9AM

## Issue(s)

Fixes #1143

## How to test

1. Go to the homepage
2. Check notification text and link matches the ticket description (link won't work until the new blog post is published on Monday morning)

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
